frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -30,6 +37,47 @@ export const MatchesList = ({tournamentId}: MatchesListProps) => {
          });
    }, [tournamentId]);
 
+   useEffect(() => {
+      if (!matchesListItems) return;
+
+      async function setMatchesInformationListDtoState(): Promise<void> {
+         if (!matchesListItems) return;
+         const returnedMatchesInformationListDto = await Promise.all(matchesListItems
+            .map(async (matchListItem) => {
+                  const matchId = tournamentId + matchListItem.matchNumber;
+                  return await getMatchInformationDto(matchId)
+               }
+            ))
+         await setMatchesInformationListDto(returnedMatchesInformationListDto);
+      }
```

Comment:
To da się zwięźlej zapisać:
```suggestion
   useEffect(() => {
      if (!matchesListItems) return;

      Promise.all(matchesListItems
          .map((matchListItem) => getMatchInformationDto(`${tournamentId} ${matchListItem.matchNumber}`)))
          .then(matchesDetails => setMatchesInformationListDto(matchesDetails))

      getTeamsListDto(tournamentId)
          .then(teamsListDto => setTeamsListDto(teamsListDto))
      
   }, [matchesListItems]);
```

Reply:
Będzie!

Reply:
@nowakprojects 
Tu widać, że jesteś SENIOREM!!!!
Niedługo też będę tak pisał!
Bardzo ładne i czytelne!
To jest to czego chcę!